### DISCUSSION TITLE
Tap to peek

### DISCUSSION THREAD

**Discussion by modestderp** [Docile-Alligator/Infinity-For-Reddit#941] [Ideas]
I would like to ask whether it is possible to implement a "tap and peek" feature?

What I am referring to is having a function which allows the user to tap and hold on a image thumbnail of the post, which then would result in a popup of the image in full resolution. The full resolution image popup would then disappear when the user lifts their finger.

**Comment by Docile-Alligator** [comment]
Good idea. I need to think about how to implement this.
